src/models/Organization.ts

```diff
@@ -66,8 +74,31 @@ const organizationSchema = new Schema(
       type: String,
       required: true,
     },
-    location: {
-      type: String,
+    address: {
```

Comment:
confirm which fields are required and add `required: true` to them

Reply:
Ok sir, I will make those two mandatory. As for the enums, I have already hardcoded an entire list of objects with country codes and country names in the admin portal for the drop down list. I found that creating an object would be more self-explanatory and better as compared to just an enum. you can find that change in the other PR. Thank you.

Reply:
@palisadoes @xoldyckk Respected sir, I have made the changes requested, and made countrycode, and city mandatory. But due to this change. Some tests are failing. I will be fixing those tests as early as possible. 

Thank You.